exercises/all-your-base/src/lib.rs

```diff
@@ -33,7 +33,7 @@ pub enum Error {
 ///
 /// Notes:
 ///  * The empty slice ( "[]" ) is equal to the number 0.
-///  * Never output leading 0 digits. However, your function must be able to
+///  * Never output leading 0 digits, unless the input number is 0, in which the output must be `[0]`. However, your function must be able to
 ///     process input with leading 0 digits.
```

Comment:
This line is really wide. 
```suggestion
///  * Never output leading 0 digits, unless the input number is 0, in which the output must be `[0]`.
///    However, your function must be able to process input with leading 0 digits.
```